pkg/scheduler/algorithmprovider/defaults/register_priorities.go

```diff
@@ -70,7 +70,7 @@ func init() {
 	)
 	// pods should be placed in the same topological domain (e.g. same node, same rack, same zone, same power domain, etc.)
 	// as some other pods, or, conversely, should not be placed in the same topological domain as some other pods.
-	scheduler.RegisterPriorityMapReduceFunction(priorities.InterPodAffinityPriority, priorities.CalculateInterPodAffinityPriorityMap, priorities.CalculateInterPodAffinityPriorityReduce, 1)
+	scheduler.RegisterPriorityMapReduceFunction(priorities.InterPodAffinityPriority, nil, nil, 1)
```

Comment:
yeah, this is because we want to keep this priority **key** registered, the logic downstream will basically look at the registered keys and configure the scheduler with plugins. This will be cleaned up later.